endoreg_db/utils/ai/multilabel_classification_net.py

```
@@ -78,10 +127,47 @@ def __init__(
+                device_hint = f"{device.type}:{device.index}" if device.index is not None else device.type
+            else:
+                device_hint = device
+
+            state_dict = load_file(path, device=device_hint)
+
+            instance = cls(
+                labels=labels,
+                model_type=model_type,
+                load_imagenet_weights=load_imagenet,
+                track_hparams=False,
+                *args,
```

Ruff (0.14.3)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)